relay-sampling/src/lib.rs

```diff
@@ -856,6 +856,10 @@ impl FieldValueProvider for DynamicSamplingContext {
                 None => Value::Null,
                 Some(ref s) => s.as_str().into(),
             },
+            "trace.replay_id" => match self.replay_id {
+                None => Value::Null,
+                Some(ref s) => Value::String(s.to_string()),
```

Comment:
nit: For consistency with the code above

```suggestion
                Some(ref s) => s.as_str().into(),
```

Reply:
I have tried that but `Uuid` has no method `as_str()`.